tests/unit/models/reddit/test_wikipage.py

```diff
@@ -1,7 +1,5 @@
 import pickle
 
-import pytest
```

Comment:
Is this import necessary?

Reply:
This has been removed in the latest commit.